MonoGame.Framework/Vector3.cs

```diff
@@ -853,6 +923,41 @@ public static void Reflect(ref Vector3 vector, ref Vector3 normal, out Vector3 r
             result.Z = vector.Z - (2.0f * normal.Z) * dotProduct;
         }
 
+        /// <summary>
+        /// Round this <see cref="Vector3"/> to a vector with the nearest integral value.
+        /// </summary>
+        public void Round()
+        {
+            X = (float)Math.Round(X);
+            Y = (float)Math.Round(Y);
+            Z = (float)Math.Round(Z);
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="Vector3"/> that contains rounded values from another vector.
+        /// </summary>
+        /// <param name="value">Source <see cref="Vector3"/>.</param>
+        /// <returns>Unit vector.</returns>
```

Comment:
This seems misleading as the return value description.  "Unit vector" typically means "a vector that has a magnitude of one".